src/partials/layout/_footer.html

```diff
@@ -21,88 +21,78 @@
+            <a class="footer-nav__link link" href="./3d-scanning.html"
               >3D Сканування</a
             >
           </li>
-          <li class="footer__nav--item">
-            <a
-              class="footer__nav--link link"
-              href="./3d-printing.html"
+          <li class="footer-nav__item">
+            <a class="footer-nav__link link" href="./3d-printing.html"
               >3D Друк</a
             >
           </li>
-          <li class="footer__nav--item">
-            <a class="footer__nav--link link" href="./order-page.html"
+          <li class="footer-nav__item">
+            <a class="footer-nav__link link" href="./order-page.html"
               >Замовити</a
             >
           </li>
-          <li class="footer__nav--item">
-            <a class="footer__nav--link link" href="./contact.html"
+          <li class="footer-nav__item">
+            <a class="footer-nav__link link" href="./contact.html"
               >Контакти</a
             >
           </li>
         </ul>
       </nav>
-      <div class="footer__social">
-        <ul class="footer__social-list list">
-          <li class="footer__social--item">
-            <a class="footer__social--link link" href="#"
-              >Instagram</a
-            >
+      <div class="footer-social">
+        <ul class="list">
+          <li class="footer-social__item">
+            <a class="footer-social__link link" href="#">Instagram</a>
           </li>
-          <li class="footer__social--item">
-            <a class="footer__social--link link" href="#">YouTube</a>
+          <li class="footer-social__item">
+            <a class="footer-social__link link" href="#">YouTube</a>
           </li>
-          <li class="footer__social--item">
-            <a class="footer__social--link link" href="#">Facebook</a>
+          <li class="footer-social__item">
+            <a class="footer-social__link link" href="#">Facebook</a>
           </li>
         </ul>
       </div>
-      <div class="footer__contact">
-        <address>
-          <p class="footer__mail-heading">Email</p>
-          <a
-            class="footer__contact-link link"
-            href="mailto:3dfreedom.info@gmail.com"
-            >3dfreedom.info@gmail.com</a
-          >
-          <p class="footer__tel-heading">Телефони</p>
-          <ul class="list">
-            <li class="footer__contact-tel-item">
-              <a
-                class="footer__contact-tel-link link"
-                href="tel:+380936432288"
-                >+38 (093) 643 22 88</a
-              >
-            </li>
-            <li class="footer__contact-tel-item">
-              <a
-                class="footer__contact-tel-link link"
-                href="tel:+380503454423"
-                >+38 (050) 345 44 23</a
-              >
-            </li>
-          </ul>
-        </address>
-      </div>
+      <address class="footer-contact">
+        <p class="footer-contact__mail--heading">Email</p>
```

Comment:
Rename footer-contact__mail--heading by BEM. BEM block: "block-name", BEM element: "block-name__element"

Reply:
fixed

Reply:
Still not by BEM - corrections necessary

Reply:
corrected